decidim-assemblies/app/views/layouts/decidim/admin/assemblies.html.erb

```diff
@@ -1,6 +1,6 @@
 <% content_for :breadcrumb_context_menu do %>
   <div class="process-title-content-breadcrumb-container-right">
-    <% if allowed_to? :create, :assembly %>
+    <% if allowed_to? :create, :assembly, assembly: parent_assembly %>
```

Comment:
Reviewing, this is a bit troubling. 
Ideally, we would do a `allowed_to? :create, :assembly, parent_assembly: parent_assembly`... but I guess that's not an easy change.

Reply:
Why should we change `assembly` to `parent_assembly` ?

Reply:
because you want to know "are you allowed to create an assembly with this parent_assembly?", as it make little sense to check if you are allowed to create an assembly with an assembly parameters.